purchase_request_to_rfq_order_approved/views/purchase_request_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<!-- Copyright 2017 Eficent Business and IT Consulting Services S.L.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<openerp>
+    <record id="purchase_request_line_search" model="ir.ui.view">
+        <field name="name">purchase.request.line.search</field>
+        <field name="model">purchase.request.line</field>
+        <field name="inherit_id" ref="purchase_request_to_rfq.purchase_request_line_search"/>
+        <field name="arch" type="xml">
+            <filter name="purchase_state_draft" position="after">
+                <filter name="purchase_state_approved"
+                        string="Purchase Approved"
+                        domain="[('purchase_state','=', 'approved')]"
+                        help="At least a PO has been approved"/>
```

Comment:
<code>help</code> attribute can be a long string. So it's nicer to read Purchase Order instead of PO.